test/AspNet.Security.OAuth.Providers.Tests/Streamlabs/StreamlabsTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AspNet.Security.OAuth.Streamlabs
+{
+    public class StreamlabsTests : OAuthTests<StreamlabsAuthenticationOptions>
+    {
+        public StreamlabsTests(ITestOutputHelper outputHelper)
+        {
+            OutputHelper = outputHelper;
+        }
+
+        public override string DefaultScheme => StreamlabsAuthenticationDefaults.AuthenticationScheme;
+
+        protected internal override void RegisterAuthentication(AuthenticationBuilder builder)
+        {
+            builder.AddStreamlabs(options => ConfigureDefaults(builder, options));
+        }
+
+        [Theory]
```

Comment:
Please add a test for each claim mapped by default in the options.

Reply:
I've added missing tests. https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/pull/553/commits/8500d738ed29df3646cdcb35a97d3a0b9ec6ace3

Sorry for taking your time.